acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/signing/KeyLoadAndSignAcceptanceTest.java

```diff
@@ -145,7 +147,7 @@ public void unusedFieldsInRequestDoesNotAffectSigning() throws JsonProcessingExc
     final Path keyConfigFile = testDirectory.resolve(configFilename + ".yaml");
     METADATA_FILE_HELPERS.createUnencryptedYamlFileAt(keyConfigFile, PRIVATE_KEY, KeyType.BLS);
 
-    setupEth2SignerMinimalWithoutAltairFork();
+    setupEth2Signer(Eth2Network.MINIMAL, SpecMilestone.PHASE0);
```

Comment:
Is there any benefit to having a BELLATRIX version of these tests as well?

Reply:
The Bellatrix version is part of BLOCK_V2 AT specifically (along with Altair).

Reply:
Also, I think all of the tests in this class are for error scenarios.